### PR TITLE
Support unlimited assets REST API changes.

### PR DESCRIPTION
**Pull request by nicholasguoalgorand**
Add endpoints related to the unlimited assets changes.

### CODE REVIEW

File: src/client/v2/algod/accountInformation.ts [hunk 1]
```diff
@@ -15,4 +15,23 @@ export default class AccountInformation extends JSONRequest {
...
+   * const address = "XBYLS2E6YI6XXL5BWCAMOA4GTWHXWENZMX5UHXMRNWWUQ7BXCY5WC5TEPA";
+   * const accountInfo = await algodClient.accountInformation(address)
+   *        .excludeInfo('all')
+   *        .do();
+   * ```
+   *
+   * @param round
+   * @category query
+   */
+  excludeInfo(exclude: string) {
```
Comment (winder) [inline src/client/v2/algod/accountInformation.ts:33]: Should this be an array, or maybe mention in the documentation that something like `created-asset,created-apps` is a valid input?

**Review by winder — CHANGES_REQUESTED** [review]
It looks like, at least, the round param has not been removed from the balances endpoint.

Have the model objects been generated? I don't see the creator field removed either.

**Comment by winder** [comment]
The new tests need to be enabled: `@unit.responses.unlimited_assets,@unit.indexer.ledger_refactoring,@unit.algod.ledger_refactoring`

**Comment by nicholasguoalgorand** [comment]
> It looks like, at least, the round param has not been removed from the balances endpoint.
> 
> Have the model objects been generated? I don't see the creator field removed either.

How do I generate the model?

File: src/client/v2/algod/models/types.ts [hunk 2]
```diff
@@ -1360,14 +1492,14 @@ export class DryrunTxnResult extends BaseModel {
 export class ErrorResponse extends BaseModel {
   public message: string;
 
-  public data?: string;
+  public data?: Record<string, any>;
```
Comment (winder) [inline src/client/v2/algod/models/types.ts:1495]: Did you update the generator to support this change?

**Review by winder — COMMENTED** [review]
One question. Changes look good, but the tests aren't passing.

**Comment by winder** [comment]
@algochoi do you know anything about the [build_and_test_chrome](https://circleci.com/gh/algorand/js-algorand-sdk/531?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link) test? It seems to be flaky recently and we're not sure whats causing it.

**Comment by algochoi** [comment]
@winder At first glance, it looks like there is some version mismatch between our chromedriver versions:

```
SessionNotCreatedError: session not created: This version of ChromeDriver only supports Chrome version 97
```

I can take a closer look later today as well...

**Review by winder — APPROVED** [review]

**Pull request merged**